src/modules/DialogOverlay.vue

```diff
@@ -5,8 +5,11 @@
     :button-close-label="buttonCloseLabel"
     @toggle-dialog="toggleDialog"
   >
+    <template #title>
+      <slot name="title">Interest-free Payment</slot>
+    </template>
     <template #header>
-      <slot name="header" />
+      <MastercardExisting />
```

Comment:
would it be better to keep the slot and pass in MastercardExisting from parent? It's not a blocker but will probably have to be tackled as part of Humm90 theme work.
![image](https://user-images.githubusercontent.com/63985172/134445172-06aded1a-56d2-4c68-93ea-bd81d34e7196.png)

Reply:
Yep, feel free to do that if you want to! This is your baby now 😁